Cargo.lock

```diff
@@ -7616,9 +7616,9 @@ dependencies = [
 
 [[package]]
 name = "parity-db"
-version = "0.4.11"
```

Comment:
I don't see any Cargo.toml change?

Reply:
It is indirect dependency

Reply:
Hm, I don't understand this. Upstream fork substrate/client/db/Cargo.toml has `parity-db = "0.4.8"`. But our repo has 0.4.11, where does this version come from?

Also, for indirect dependency, usual way is to change the call site (upstream fork in this case) to use the desired version, so it will automatically get picked by mono repo. Not sure why we are changing the Cargo.lock directly

Reply:
Cargo will always install newest semver-compatible version unless you specify something like `parity-db = "=0.4.8"` to fix a specific version. I just upgraded it with `cargo update -p parity-db` to newer semver-compatible version.

Reply:
I see, ok. I have never used this method before, always go through the Cargo.toml (like in this case Cargo.toml and Cargo.lock point to different versions)